minicluster/src/main/java/alluxio/master/LocalAlluxioMaster.java

```diff
@@ -139,6 +139,7 @@ public void stop() throws Exception {
    * @throws Exception if master thread cannot be interrupted
    */
   public void kill() throws Exception {
+    mAlluxioMaster.stop();
```

Comment:
Yeah, I noticed that `kill()` didn't actually kill anything, and masters never failed over even after more than 30 seconds. I don't know if `kill` ever worked like we want it to work.